core/server/proxy/src/main/java/alluxio/web/ProxyWebServer.java

```diff
@@ -102,6 +106,12 @@ public ProxyWebServer(String serviceName, InetSocketAddress address,
         .register(S3RestExceptionMapper.class);
 
     mFileSystem = FileSystem.Factory.create(Configuration.global());
+    long rate =
+        Configuration.getLong(PropertyKey.PROXY_S3_GLOBAL_READ_RATE_LIMIT_MB) * Constants.MB;
+    if (rate <= 0) {
+      rate = Long.MAX_VALUE;
+    }
+    mGlobalRateLimiter = RateLimiter.create(rate);
```

Comment:
I expect we have a switch. if user don't need this feature, they do not need to initalize the Limiter

Reply:
It is a good idea, we shoud check the value, if users not set rate limiter, we shoud fall back to original implements.